addons/website_forum/static/src/js/website_forum.js

```diff
@@ -129,7 +129,11 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
                 recordInfo: {
                     context: self._getContext(),
                     res_model: 'forum.post',
-                    res_id: +window.location.pathname.split('-').pop(),
+                    // Id is retrieved from URL, which is either:
+                    // - /forum/name-1/post/something-5
+                    // - /forum/name-1/post/something-5/edit
+                    // TODO: Make this more robust.
+                    res_id: +window.location.pathname.split('-').slice(-1)[0].split('/')[0],
```

Comment:
Should this be a stable fix? (can merge like that for now of course but to be sure to understand)

Reply:
Probably, even tho I don't really know for what else is the `res_id` used except for the image upload (which is not possible in lower version)
I'd leave it like that and just adapt the TODO to have something more robust in 16.0 or master later

Reply:
(Should be used to list the right images when re-accessing the record (in the media dialog), we'll see later